src/db2wish.c

```diff
@@ -16,6 +16,10 @@
 #include "tk.h"
 #include "db2tcl.h"
 
+#if TK_MAJOR_VERSION >= 9
+int Tcl_AppInit(Tcl_Interp *);
```

Comment:
According to the docs, this prototype should be provide by `<tcl.h>`.  Do we need to include that header?